cmd/kubeadm/app/util/version.go

```diff
@@ -83,3 +82,49 @@ func KubernetesVersionToImageTag(version string) string {
 	allowed := regexp.MustCompile(`[^-a-zA-Z0-9_\.]`)
 	return allowed.ReplaceAllString(version, "_")
 }
+
+// KubernetesIsCIVersion checks if user requested CI version
+func KubernetesIsCIVersion(version string) bool {
+	subs := kubeBucketPrefixes.FindAllStringSubmatch(version, 1)
+	if len(subs) == 1 && strings.HasPrefix(subs[0][2], "ci") {
```

Comment:
FindAllStringSubmatch for regexp will return list of elements based on amounts of brackets specified in regular expression. Thus, if amount of matches is non-zero, every match will have at least 3 elements. I can add additional len check, but it will be always true.